srv/wordpress/wp-content/plugins/acore-wp-plugin/src/components/AdminPanel/Settings.view.php

```diff
@@ -394,6 +427,18 @@ public function getElunaSettingsRender() {
                     },
                 })
             });
+            jQuery('#eluna_recruit_a_friend').on('change', function() {
+                if (this.value == '1') {
+                    jQuery('#eluna_raf_config').show();;
+                } else {
+                    jQuery('#eluna_raf_config').hide();
+                }

```

Comment:
could we use `jQuery('#eluna_raf_config').toggle();` ?

moreover tehre are two `;;` in `.show();;`